lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -142,14 +145,14 @@ class UnusedBytes extends Audit {
       networkNode.record._transferSize = originalTransferSize;
     });
 
-    const savingsOnTTI = Math.max(
-      Interactive.getLastLongTaskEndTime(simulationBeforeChanges.nodeTimings) -
-        Interactive.getLastLongTaskEndTime(simulationAfterChanges.nodeTimings),
-      0
-    );
+    const savingsOnLoad = options.includeLoad ?
```

Comment:
how about we just default to this? no option needed.

Reply:
well that's effectively what's happening with the default option above

Reply:
sure, just pitching the idea of removing this condition entirely. *always* use `savingsOnload`.

Reply:
maybe agree with the above. It's not clear why `offscreen-images` is different and would be the only one to opt out

Reply:
because this function takes how many bytes that can be removed from each resource and simulates the graph if they were removed.

in offscreen images, we can't actually remove the bytes from being loaded at all. i.e. the result of this function with `includeLoad` is no longer "defer offscreen images", but "delete all offscreen images". restricting the computation to savings on TTI preserves the contract of "defer offscreen images" because it's saying how much earlier could you push TTI if you loaded all this junk *after* TTI

Reply:
ok, makes sense. Maybe `savingsOnOverallLoad` or something for the variable name to make it clear it's our handwavey overall load numbers and not `onload`?

Reply:
done